### ISSUE TITLE
How are sounds found

### ISSUE THREAD

**Issue by z3t0** [deathcap/voxel-sfx#8]
How does artpacks.getSound work? I am trying to play a sound called 'background.ogg' inside assets/minecraft/sounds/random but cant get it to work

**Comment by deathcap** [comment]
Sounds are loaded via the https://github.com/deathcap/voxel-artpacks instance of https://github.com/deathcap/artpacks - cascading from the loaded packs (used for both textures and sounds, and potentially other kinds of artwork in the future).

https://github.com/deathcap/artpacks/blob/master/artpacks.coffee#L171

Are you able to get it to work with: game.plugins.get('voxel-sfx').play('random/background')? If not try stepping through the debugger into the artpacks module, it should resolve 'random/background' to a path within the zip archives.

**Comment by z3t0** [comment]
yeah I got it working, thanks! check github,com/z3t0/voxel-background-music
I'm updating the voxelmetaverse now, take a look at my PR for voxel-sfx since its needed for the background plugin

**Comment by z3t0** [comment]
my PR adds the loop option, which is needed for background music
